configs/analysis_configs/cytotable_convert.yaml

```diff
@@ -0,0 +1,11 @@
+cytotable_convert:
+  params:
+    dest_datatype: parquet
+    source_datatype: null
+    concat: True
+    join: True
```

Comment:
This is probably beyond scope of `cytosnake`, but can you specify non-canonical compartments? (e.g. anything beyond Cells, Cytoplasm, and Nuclei)

Reply:
This will depend on `CytoTable`'s capability. The config files will take anything as long as the parameters are valid in `CytoTable`'s `convert` workflow. Therefore, if there is support for non-canonical compartments within `CytoTable` then it should work? 

Looking at the documentation, the `join` [parameter](https://github.com/cytomining/CytoTable/blob/f336d26baf96b8b3d9ed6ca07f617895030bf038/cytotable/convert.py#L861-L862) only takes a boolean. 

Correct me if I am wrong @d33bs.This will depend on `CytoTable`'s capability. The config files will take anything as long as the parameters are valid in `CytoTable`'s `convert` workfow. Therefore, if there is support for non-cononical compartments within `CytoTable` then it should work? 

Looking at the documentation, the `join` [parameter](https://github.com/cytomining/CytoTable/blob/f336d26baf96b8b3d9ed6ca07f617895030bf038/cytotable/convert.py#L861-L862) only takes a boolean. 

Correct me if I am wrong @d33bs.